### DISCUSSION TITLE
Site Not Found [404] after changing repo visibility from public to private on gh-pages. No solution working, Please help!

### DISCUSSION THREAD

**Discussion by TejasBishnoi21** [community/community#88666] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I had deployed my portfolio on gh-pages long ago and it was working fine untill the visibility of the repo was set to public but yesterday when I made it private and deployed the changes using `npm run deploy` the website got crashed and since then it says "Site not found - 404".
Solutions I've tried - 
1. Pushed some changes in master branch 
2. Renamed the repository's name to something else and renamed back to original.

Nothing seems to be working out for me, please help!

**Comment by AidenJamesYt** [comment]
After changing the visibility of your GitHub repository from public to private, you encountered a "Site not found - 404" error when attempting to deploy your portfolio on GitHub Pages. To address this, first, confirm that your GitHub Pages settings are correctly configured in the repository settings, specifying the appropriate source branch. Try redeploying your portfolio using the command npm run deploy. Allow some time for GitHub Pages to rebuild the site. Clear your browser cache or try accessing the site in an incognito/private browsing window. Double-check the repository name in your deployment script, ensuring it matches the current repository name. Verify that the deploying account has the necessary permissions for GitHub Pages in the private repository. Review deployment logs for any error messages, and update npm packages with npm install -g npm and npm install.

**Comment by Andrkenya** [comment]
Hi @TejasBishnoi21

Please ensure the repo your publishing is public. Try to create a new public repo and deploy the same project or another project you will confirm the faults..
